### PR TITLE
Don't show chat widget context menu for filetree

### PR DESCRIPTION
**Pull request by joyceerhl**

### CODE REVIEW

**Comment by joyceerhl** [comment]
cc @roblourens is there a better way to avoid showing the chat copy actions for the filetree within a chat response?

**Review by roblourens — DISMISSED** [review]
The file tree widget could intercept this event. Probably at some point it would have its own context menu. "Open to the side", "copy path" maybe there is something else useful

File: src/vs/workbench/contrib/chat/browser/chatWidget.ts [hunk 1]
```diff
@@ -355,6 +355,17 @@ export class ChatWidget extends Disposable implements IChatWidget {
 		e.browserEvent.preventDefault();
 		e.browserEvent.stopPropagation();
 
+		if ('target' in e.anchor) {
+			const { className } = e.anchor.target;
+			// Don't show the context menu for file trees in the response
+			if (className.includes('codicon-tree-item-expanded') ||
```
Comment (roblourens) [inline src/vs/workbench/contrib/chat/browser/chatWidget.ts:361]: You can probably figure out whether this element is below whatever class marks the file tree that we know about? These might hit on some other random sub element at some point

Comment (joyceerhl) [reply src/vs/workbench/contrib/chat/browser/chatWidget.ts]: 🤡 I tried `preventDefault` but forgot `stopPropagation`, that did the trick

**Review by TylerLeonhardt — APPROVED** [review]

**Pull request merged**